stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -77,15 +76,20 @@ def make_procurement(self):
             )
             # Run procurement
             try:
-                pg_obj.run(
-                    item.orderpoint_id.product_id,
-                    item.qty,
-                    item.uom_id,
-                    item.orderpoint_id.location_id,
-                    item.orderpoint_id.name,
-                    item.orderpoint_id.name,
-                    values,
+                procurements.append(
+                    pg_obj.Procurement(
+                        item.orderpoint_id.product_id,
+                        item.qty,
+                        item.uom_id,
+                        item.orderpoint_id.location_id,
+                        item.orderpoint_id.name,
+                        item.orderpoint_id.name,
+                        item.orderpoint_id.company_id,
+                        values,
+                    )
                 )
+                pg_obj.run(procurements)
+                procurements.clear()
```

Comment:
This is inefficient. Odoo added the `Procurement` namedtuple exactly for this, to be able to do several procurements with one call to `run` method. Please, have a look [at this](https://github.com/OCA/manufacture/blob/13.0/mrp_multi_level/wizards/mrp_inventory_procure.py#L89) and follow the same approach: first construct a list of procurements and after that call `run` method just once.